test/harness/jpype/attr/Test1.java

```diff
@@ -62,11 +62,11 @@ public static String testStaticHolder(Holder h)
 	
 	public String stringValue = "Foo";
 	public char charValue = 'a';
-	public static Object objectValue= new Integer(234);
+	public static Object objectValue= (int)234;
```

Comment:
why do you need to trigger autoboxing? The result is still a static object

Reply:
For reasons that are beyond me Integer(int) is marked deprecated after Java 9.   Instead they want you use valueOf. But that doesn't exist on the earlier version.  So the best I could do to fit both is autoboxing.   It seems really silly to me.

Reply:
yes, that's hilarious